File: scoreboard-server/ScoreboardServer/Models/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ScoreboardServer.Models
{
    public class Player
    {
        public int Id { get; set; }
        public int Number { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PhotoPath { get; set; }
        public int TeamId { get; set; }
        public List<Stats> Stats { get; set; }
        public string ApplicationUserId { get; set; }
    }
}

File: scoreboard-client/src/app/models/player.ts
import {Stats} from './stats';

export class Player {
  id: number;
  number: number;
  name: string;
  surname: string;
  fgm = 0;
  fga = 0;
  ftm = 0;
  fta = 0;
  fgm3 = 0;
  fga3 = 0;
  fg = 0;
  ft = 0;
  fg3 = 0;
  pf = 0;
  reb = 0;
  ast = 0;
  stl = 0;
  blk = 0;
  to = 0;
  stats: Stats[];
  teamId: number;
}
